projects/plugins/jetpack/extensions/blocks/podcast-player/attributes.js

```diff
@@ -17,6 +17,10 @@ export default {
 		type: 'string',
 		validator: isUrl,
 	},
+	selectedEpisodes: {
```

Comment:
_minor_: Not 100% convinced about the attribute name. I think, in the near future, we'd like to implement an episodes selector and stores the selected one by the user in a new block attribute. so I think in something like `episodesList` and `selectedEpisode`.

Reply:
> I think, in the near future, we'd like to implement an episodes selector and stores the selected one by the user in a new block attribute

I think this is what we'd use this attribute for. At the moment we're setting the value based on the Anchor flow, but we'll be able to expose a UI to allow someone to select the episodes themselves.